akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ProducerControllerImpl.scala

```diff
@@ -360,7 +359,7 @@ private class ProducerControllerImpl[A: ClassTag](
         else Vector.empty // no resending, no need to keep unconfirmed
 
       if (s.currentSeqNr == s.firstSeqNr)
-        timers.startTimerWithFixedDelay(ResendFirst, ResendFirst, 1.second)
+        timers.startTimerWithFixedDelay(ResendFirst, delay = settings.durableQueueResendFirstInterval)
```

Comment:
This scheduling could be simplified because default key is a msg argument. If you did it intentionally I will revert this commit.

Reply:
good